main.tf

```diff
@@ -81,11 +81,11 @@ resource "aws_rds_cluster" "this" {
 resource "aws_rds_cluster_instance" "this" {
   count = var.replica_scale_enabled ? var.replica_scale_min : var.replica_count
 
-  identifier                      = "${var.name}-${count.index + 1}"
+  identifier                      = try(var.instances_parameters[count.index].instance_name,"${var.name}-${count.index + 1}")
```

Comment:
`try` is experimental and should not be used

Reply:
@antonbabenko I don't see [try](https://www.terraform.io/docs/configuration/functions/try.html) function to be experimental. 
Actually, it seems to be pretty clear use case for `try` if we want to keep `instance_parameters` optional.
Quote from docs: `The try function is intended only for concise testing of the presence of and types of object attributes`
It supported since `0.12.20`, so that to be stated in release notes and/or readme.

What do you think?

Reply:
Right, I was thinking about this - https://www.terraform.io/docs/configuration/variables.html#custom-validation-rules - but if I understand it correctly, it is only for `can`, but not for `try`. In any case, `try` is rather new to put it in reusable modules. Let's be an old-fashion here for the next month or so.

Reply:
@antonbabenko Created a separate PR #132 
Not sure how can I update the current one